package.json

```diff
@@ -6,7 +6,7 @@
   "publisher": "mrmlnc",
   "license": "MIT",
   "engines": {
-    "vscode": "^1.10.0"
+    "vscode": "^1.15.0"
```

Comment:
Please, set required version of VS Code to `^1.16.0`, because I support only two versions of the editor: current and previous.

Reply:
Done!